File: src/main/java/com/ids1024/whitakerswords/WhitakersSettingsFragment.java
package com.ids1024.whitakerswords;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

public class WhitakersSettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences, rootKey);
    }
}

File: src/main/kotlin/WhitakersSettingsFragment.kt
package com.ids1024.whitakerswords

import android.os.Bundle
import android.support.v7.preference.PreferenceFragmentCompat

class WhitakersSettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences, rootKey)
    }
}
